Routes/Product listing page/product.html

```diff
@@ -0,0 +1,343 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Shoe-kart | Home</title>
+    <link rel="stylesheet" href="/Stylesheets/main.css">
+    <link rel="stylesheet" href="https://maximus-ui.netlify.app/Stylesheets/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+</head>
+
+<body>
+    <div class="page-container">
+        <nav class="main-nav flex-spacearound">
+            <div class="nav-category">
+                <h1 class="brand-name">Shoe-Kart</h1>
+                <a class="category-link men-categ-link" href="#">
+                    <h3> MEN </h3>
+                </a>
+                <a class="category-link women-categ-link" href="#">
+                    <h3> WOMEN </h3>
+                </a>
+                <a class="category-link kids-categ-link" href="#">
+                    <h3> KIDS </h3>
+                </a>
```

Comment:
Don't put heading tags inside anchor tags, it should be the other way around ( `a` inside `h3` )

Reply:
sure will keep in mind from next time